harness/determined/pytorch/deepspeed/_deepspeed_trial.py

```diff
@@ -898,23 +847,14 @@ def _validate(self, searcher_op: Optional[core.SearcherOperation] = None) -> Dic
             # Get best validation before reporting metrics.
             best_validation_before = self.core_context.train.get_experiment_best_validation()
 
-            self.core_context.train.report_validation_metrics(self.state.batches_trained, metrics)
-        searcher_metric = None
+            # We report "batch" and "epoch" only if these keys are not already reported in user
+            # metrics.
+            metrics["batches"] = metrics.get("batches", self.state.batches_trained)
+            metrics["epochs"] = metrics.get("epochs", self.state.epochs_trained)
 
-        # Report searcher status.
-        if self.is_chief and searcher_op:
-            if self.local_training:
-                searcher_length = self.max_length
-            else:
-                searcher_length = pytorch.TrainUnit._from_searcher_unit(
-                    searcher_op.length, self.searcher_unit, self.global_batch_size
-                )
-            if self.searcher_metric_name:
-                searcher_metric = self._check_searcher_metric(metrics)
-
-            assert searcher_length
-            if self._steps_until_complete(searcher_length) < 1 and not searcher_op._completed:
-                searcher_op.report_completed(searcher_metric)
+        self.core_context.train.report_validation_metrics(
```

Comment:
We only reported these if `self.is_chief` was true, is this fine?

Reply:
hah, my bad 🤦‍♀️  was copying code and thought i got the indents right but i definitely didn't. nice catch.